Create sysbox sample (#327)
EngFlow/example: docker/sysbox/README.md

# Preparing your code base for Remote Execution

This is an example project that validates Sysbox setup for your EngFlow cluster
works correctly.

Use as companion to https://docs.engflow.com/re/client/sysbox.html

Usage:

> - Follow instructions in https://docs.engflow.com/re/client/sysbox.html to generate

Reviewer: I think this creates a bit of confusion. Sysbox != DinD. Please differentiate the 2 concepts more.

Author: Modified the text to clarify a bit more. Let me know if this looks good.

# Preparing your code base for Remote Execution

This is an example project that validates Sysbox setup for your EngFlow cluster
works correctly.

Use as companion to https://docs.engflow.com/re/client/sysbox.html

Usage:

- Follow instructions in https://docs.engflow.com/re/client/sysbox.html to generate

>  a container tha runs with `sysbox` runtime which enables running nested docker containers.

Reviewer: ```suggestion
 a container tha uses `sysbox` to run nested docker containers.
```
